scripts/run_stubtest.py

```diff
@@ -0,0 +1,96 @@
+import os
+from pathlib import Path
+import sys
+import tempfile
+import warnings
+
+from mypy import stubtest
+
+import pandas as pd
+
+# fail early if pandas is not installed
+if "dev" not in getattr(pd, "__version__", ""):
+    # 'succeed' but print warning
+    warnings.warn("You need to install the development version of pandas")
```

Comment:
It might be nice for this to be a hard error if running in CI, so this doesn't silently break

Reply:
@mroeschke is a environment variable that is only available when running on the CI? Could then fail silently only when this variable is not present.

Reply:
Yup `PANDAS_CI = 1`. In Python, you can check this with `pandas.compat.is_ci_environment()`